whitepaper/dsnp_whitepaper.tex

```diff
@@ -38,6 +48,10 @@
 
 \maketitle
 
+\standalonefootnote{\copyright\, 2020 Liberty 3.0 LLC.\\
```

Comment:
@sbendar Should this be Unfinished LLC or perhaps something else?

Reply:
Ah, yeah, thanks @wilwade -- I was taking a guess, but obviously please adjust as needed.

Reply:
(Most of my time went to LaTeX research, as you might guess :-) .)

Reply:
Confirmed that we will go with UnfinishedLabs LLC until we transfer the rights to the DNSP foundation.